### PR TITLE
ui: improved the looks of pagination

### LINKED ISSUE #124: ui: improve the looks of pagination
Icon styling could be improved by specifying the icon names like in this [example](https://react.semantic-ui.com/addons/pagination/#usage-shorthand)

### PR DESCRIPTION
**Pull request by audrium**
closes #124

### CODE REVIEW

**Comment by audrium** [comment]
Workflow List (Old):


Workflow List (New):


Workflow Files (Old):


Workflow Files (New):

File: reana-ui/src/pages/workflowList/WorkflowList.js [hunk 1]
```diff
@@ -92,8 +93,6 @@ function Workflows() {
               interval.current = null;
```
Comment (mvidalgarcia) [inline reana-ui/src/pages/workflowList/WorkflowList.js:93]: I guess you can also remove the `defaultActivePage` here (4 lines above).

**Review by mvidalgarcia — APPROVED** [review]
LGTM 👍

File: reana-ui/src/pages/workflowList/WorkflowList.js (hunk 1 above)
Comment (audrium) [reply reana-ui/src/pages/workflowList/WorkflowList.js:93]: Yep, I tried that, it seems that `semantic-ui-react` does not set this value by default, so removing it results in an error:
```Received NaN for the `value` attribute. If this is expected, cast the value to a string.```.
So it ether have to be `defaultActivePage` or `activePage` prop set for this to work correctly.

Comment (mvidalgarcia) [reply reana-ui/src/pages/workflowList/WorkflowList.js:93]: Ah ok, I see 👍

**Pull request merged**